aerogear/mobile-client-service
# Added routes for creating, triggering and deleting builds

# Motivation

https://issues.jboss.org/browse/AEROGEAR-7911
https://issues.jboss.org/browse/AEROGEAR-7915
https://issues.jboss.org/browse/AEROGEAR-7914

# Verification steps

1. setup local OpenShift instance with mobile-core
2. provision mobile ci/cd service
3. fetch and checkout this PR
4. `export NAMESPACE=<target-project>`
5. `export KUBERNETES_CONFIG=$HOME/.kube/config`
6. `make serve`
7. create cordova mobile app using mobile developer console (or via openshift) with name `test`
8. test android and git basic auth
  a. save this to a file https://gist.github.com/jhellar/e774e7506bd10d19f109831753d5266e
  b. `npm install node-fetch`
  c. `node test1.js`
9. test ios and git ssh auth
  a. save this to a file https://gist.github.com/jhellar/9cc3d1cf37512ae344f6e9cc8d5fbbf4
  b. `npm install node-fetch`
  c. `node test2.js`
10. check response
11. check that buildconfig and secrets have been created in openshift
